frontend/app/src/components/defi/Borrowing.vue

```diff
@@ -7,50 +7,72 @@
   <v-container v-else>
     <v-row>
       <v-col cols="12">
-        <h2>MakerDAO Vaults</h2>
+        <h2>Collateralized Loans</h2>
       </v-col>
     </v-row>
     <v-row>
-      <v-col>
-        <stat-card title="Total collateral locked">
-          <amount-display
-            :value="makerDAOVaultSummary.totalCollateralUsd"
-            show-currency="symbol"
-            fiat-currency="USD"
-          ></amount-display>
-        </stat-card>
-      </v-col>
-      <v-col>
-        <stat-card title="Total debt">
-          <amount-display
-            :value="makerDAOVaultSummary.totalDebt"
-            asset="DAI"
-          ></amount-display>
-        </stat-card>
+      <v-col cols="12">
+        <stat-card-wide cols="two">
```

Comment:
Is that a our own property?

Reply:
Yes, it controls how many "cells" the stat card is wide, the default is three.